Dockerfile

```diff
@@ -41,4 +50,11 @@ CMD if [ ${APP_ENV} = production ]; \
     fresh -c .environment/fresh_runner.conf; \
     fi
 
+CMD if [ ${REMOTE_DEBUGGING} = true ]; \
```

Comment:
This is really nice, I've yet to really play with debugging in Go (LOL my code JUST WORKS!), but seriously, probably a great addition. I'll try and play with it soon.

Reply:
lol, yeah that debugger works pretty well

Reply:
This didn't dawn on me until today, but this PROBABLY won't work! You can't have two commands in a Dockerfile. If you want to do this, maybe take a look at: https://docs.docker.com/engine/admin/multi-service_container/